dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -237,6 +237,11 @@
     @ApiModelProperty(value = "The user-specified files for the version.")
     private List<String> userFiles = new ArrayList<>();
 
+    @Column(columnDefinition = "varchar")
+    @Convert(converter = UserImageReferencesConverter.class)
+    @ApiModelProperty(value = "The user-specified image references for the version.")
```

Comment:
A lot of the fields don't have it, but we should also add the openapi equivalent, `@Schema`